portal-frontend/src/app/features/edit-submission/edit-submission.component.ts

```diff
@@ -197,6 +199,9 @@ export class EditSubmissionComponent implements OnInit, OnDestroy, AfterViewInit
         if (this.pofoProposalComponent) {
```

Comment:
same as previous, just extracting into function will be good

Reply:
Good call, done!